.travis.yml

```diff
@@ -2,20 +2,21 @@ language: python
 python:
   - "3.4"
   - "3.5"
-#  - "3.5-dev" # 3.5 development branch
-#  - "nightly" # currently points to 3.6-dev
+  - "3.6"
 # command to install dependencies
 install: 
-    - pip install -r requirements.txt
-    - pip install -r requirements-dev.txt
-    - pip install -r requirements-docs.txt
+  - pip install -r requirements.txt -r requirements-dev.txt
```

Comment:
@cemsbr just to point out, considering our discussion below.

You said that if someone want to develop the project, then he/she needs to install de `requirements-dev.txt`. And if someone wants to contribute with the documentation, then he/she needs to install the `requirements-docs.txt`.

So, despite the person choice, on travis.yml we should install both requirements, shouldn't we?